<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_horas_clases</class>
 <widget class="QDialog" name="frm_horas_clases">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insertar Horas Clases</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/new/prefix1/13.png</normaloff>:/new/prefix1/13.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="commandLinkButton">
       <property name="text">
        <string>Aceptar</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/new/prefix1/iconos de menu/Sin título-2.png</normaloff>:/new/prefix1/iconos de menu/Sin título-2.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="commandLinkButton_2">
       <property name="text">
        <string>Cerrar</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/new/prefix1/iconos de menu/Sin título-1.png</normaloff>:/new/prefix1/iconos de menu/Sin título-1.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Calculo :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>EF :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Fisica :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Historia :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>EP :</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="LE_frmHoras_calculo"/>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_frmHoras_EF"/>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_frmHoras_Fisica"/>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_frmHoras_Historia"/>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_frmHoras_EP"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nombre del Gupo :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CB_frmHoras_grupo"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="img.qrc">:/new/prefix1/índice.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>commandLinkButton_2</sender>
   <signal>clicked()</signal>
   <receiver>frm_horas_clases</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
